README.md

```diff
@@ -50,6 +50,25 @@ Send a test HTTP request to the kickstart backend service:
 
     curl http://127.0.0.1/kickstart/
 
+## Smarthost setting discovery
+
+Some configuration settings, like the smarthost setup, cannot be obtained
+by the `configure-module` action: they are discovered by looking to some
```

Comment:
> cannot be obtained by the `configure-module` action:

This is not totally correct, some settings can be obtained by the `configure-module` but they can change after the execution.

Reply:
I'd replace `be obtained` with `be kept up-to-date`.

Do you agree @Amygos ?